contracts/utils/SAVAXDepositHelper.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import {ISAVAX} from "../interfaces/ISAVAX.sol";
+import {IRibbonVault} from "../interfaces/IRibbonVault.sol";
+
+contract SAVAXDepositHelper {
+    ISAVAX public sAVAX;
+    IRibbonVault public sAVAXVault;
+
+    constructor(address _sAVAX, address _sAVAXVault) {
+        require(_sAVAX != address(0), "!sAVAX");
+        require(_sAVAXVault != address(0), "!sAVAXVault");
+
+        sAVAX = ISAVAX(_sAVAX);
+        sAVAXVault = IRibbonVault(_sAVAXVault);
+    }
+
+    function deposit() external payable {
+        uint256 sAVAXAmount = sAVAX.submit{value: msg.value}();
+        sAVAX.approve(address(sAVAXVault), sAVAXAmount);
```

Comment:
sAVAX's interface would need this, consider adding

```
    function approve(address _recipient, uint256 _amount) external returns (bool);
```

to the `import {ISAVAX} from "../interfaces/ISAVAX.sol";` interface

Reply:
`approve()`'s definition will be inherited from `IERC20` so no need to add